Source/Controllers/Other/SQLiteHistoryManager.swift

```diff
@@ -6,9 +6,9 @@
 //  Copyright © 2020 Sequel-Ace. All rights reserved.
 //
 
+import Firebase
```

Comment:
Are these just whitespace/linting changes to SQLiteHistoryManager? If so, these should probably be a separate PR - makes the history clearer and easier to follow. Regardless of if these are just whitespace changes or real changes, they seem unrelated to the bookmarks changes